packages/storefront-lwc/src/modules/commerce/productDetail/productDetail.js

```diff
@@ -168,4 +168,15 @@ export default class ProductDetail extends LightningElement {
             });
         }
     }
+
+    connectedCallback() {
+        window.dispatchEvent(
```

Comment:
I don't think this is the way to go. Essentially we will need to remember to add a new CustomEvent for every future page that might need to clear the header text. It would be better to clear the header text when this component unmounts inside a `disconnectedCallback` method

Reply:
Good point.  I'll make the correction. Thank you!